src/imitation/scripts/common/expert.py

```diff
@@ -43,3 +43,19 @@ def config_hook(config, command_name, logger):
 @expert_ingredient.capture
 def get_expert_policy(venv, policy_type, loader_kwargs, common):
     return serialize.load_policy(policy_type, venv, **loader_kwargs)
+
+
+@expert_ingredient.capture
+def download_expert_rollouts(rollout_path, loader_kwargs):
```

Comment:
Does this get used anywhere other than in `common/demonstrations.py`? If not, why is it here rather than in `demonstrations.py`?

Reply:
I've defined it in `expert.py` so that we can use the [`config_hook`](https://github.com/HumanCompatibleAI/imitation/blob/2f7e4835a061516a394756a5874fc3b51faec48d/src/imitation/scripts/common/expert.py#L20) in expert that provides the default `loader_kwargs` with `organization_name` and `env_name`.

Reply:
Moved it to `demonstrations.py`.